plugins/node/opentelemetry-instrumentation-aws-sdk/src/aws-sdk.ts

```diff
@@ -503,6 +505,7 @@ export class AwsInstrumentation extends InstrumentationBase<typeof AWS> {
                   const normalizedResponse: NormalizedResponse = {
                     data: response.output,
                     request: normalizedRequest,
+                    requestId: requestId,
```

Comment:
If I remember correctly, the response sometimes only includes the `extendedRequestId` and lacks the `requestId` field. not sure what is the difference between them and when each one is used.
Just verifying that you tested it against aws and found that the request id is indeed the value to use for lambda client.
@NathanielRN @willarmiros - any insights on this maybe?

Reply:
Extended request ID is only included for S3 requests - at that I'm not even sure if the extended ID is included in responses or only errors.

See: AWS SDK v2 [error](https://github.com/aws/aws-sdk-js/blob/307e82673b48577fce4389e4ce03f95064e8fe0d/lib/error.d.ts#L44) vs. [response](https://github.com/aws/aws-sdk-js/blob/master/lib/response.js) (which has no extended ID field)

Reply:
So my tests were probably around S3 in V3 and that's why I saw extended id instead of request id. Thanks for clarifying :)